packages/fuels-accounts/src/lib.rs

```diff
@@ -11,12 +11,13 @@ pub mod wallet;
 
 #[cfg(feature = "std")]
 pub use account::*;
+#[cfg(feature = "std")]
+pub mod kms;
```

Comment:
I'd feature gate it behind a `kms_signer` flag so we don't pull in a ton of `aws-*` deps by default.